src/webots/gui/WbGuiApplication.cpp

```diff
@@ -479,8 +479,8 @@ bool WbGuiApplication::setup() {
   WbSysInfo::initializeOpenGlInfo();
   WbWrenOpenGlContext::doneWren();
 
-  if (showGuidedTour)
-    mMainWindow->showGuidedTour();
+  if (true) //(showGuidedTour)
```

Comment:
I guess you changed this for debugging purposes, and the original code should be restored.

Reply:
Yes I will restore it as soon as the review is done, it was just to make it easier for the reviewer.

Reply:
Yes, but because of that line, the tests are failing. I will fix at least the clang-format problem...